Rmagick 2.13.4 compile error

Mac OSX Yosemite
Rails 4.2.0.beta4
rake, version 10.4.0
Version: ImageMagick 6.8.9-8 Q16 x86_64  (via macports)

```
sudo bundle update
Updating git://github.com/rails/rails.git
Updating git://github.com/activeadmin/activeadmin.git
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Using rake 10.4.0
Using i18n 0.7.0.beta1 (was 0.6.11)
Using json 1.8.1
Using minitest 5.4.3
Using thread_safe 0.3.4
Using tzinfo 1.2.2
Using activesupport 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using builder 3.2.2
Using erubis 2.7.0
Using mini_portile 0.6.1
Using nokogiri 1.6.4.1
Using rails-deprecated_sanitizer 1.0.3
Using rails-dom-testing 1.0.5
Using loofah 2.0.1
Using rails-html-sanitizer 1.0.1
Using actionview 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using rack 1.6.0.beta (was 1.5.2)
Using rack-test 0.6.2
Using actionpack 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using globalid 0.3.0
Using activejob 4.2.0.beta4 from git://github.com/rails/rails.git (at master)
Using mime-types 2.4.3
Using mail 2.6.3
Using actionmailer 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using arbre 1.0.2
Using sass 3.2.19
Using thor 0.19.1
Using bourbon 3.2.3
Using coffee-script-source 1.8.0
Using execjs 2.2.2
Using coffee-script 2.3.0
Using railties 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using coffee-rails 4.0.1
Using formtastic 3.1.2
Using formtastic_i18n 0.1.1
Using has_scope 0.6.0.rc
Using responders 1.1.2
Using inherited_resources 1.5.1
Using jquery-rails 4.0.0 (was 3.1.2)
Using jquery-ui-rails 5.0.2
Using kaminari 0.16.1
Using activemodel 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using arel 6.0.0 (was 5.0.1.20140414130214)
Using activerecord 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using bundler 1.7.7
Using hike 1.2.3
Using multi_json 1.10.1
Using tilt 1.4.1
Using sprockets 2.12.3
Using sprockets-rails 2.2.1
Using rails 4.2.0.beta4 (was 4.1.6) from git://github.com/rails/rails.git (at master)
Using polyamorous 1.1.0
Using ransack 1.5.1
Using sass-rails 4.0.5
Using activeadmin 1.0.0.pre from git://github.com/activeadmin/activeadmin.git (at master)
Using bcrypt 3.1.9
Using carrierwave 0.10.0
Using orm_adapter 0.5.0
Using warden 1.2.3
Using devise 3.4.1
Using jbuilder 2.2.5
Using pg 0.17.1
Using protected_attributes 1.0.8
Using rails_serve_static_assets 0.0.2
Using rails_stdout_logging 0.0.3
Using rails_12factor 0.0.3
Using rdoc 4.1.2

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -r ./siteconf20141125-30451-c8v6p2.rb extconf.rb 
checking for Ruby version >= 1.8.5... yes
checking for xcrun... yes
checking for Magick-config... yes
checking for ImageMagick version >= 6.4.9... yes
checking for stdint.h... yes
checking for sys/types.h... yes
checking for wand/MagickWand.h... yes
checking for snprintf() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for AcquireImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for AffinityImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for AffinityImages() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for AutoGammaImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for AutoLevelImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for BlueShiftImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for ColorMatrixImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for ConstituteComponentTerminus() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for DeskewImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for DestroyConstitute() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for EncipherImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for EqualizeImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for EvaluateImages() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for FloodfillPaintImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for FunctionImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for GetAuthenticIndexQueue() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for GetAuthenticPixels() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for GetImageAlphaChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for GetMagickFeatures() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for GetVirtualPixels() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for LevelImageColors() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for LevelColorsImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for LevelizeImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for LiquidRescaleImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for MagickLibAddendum() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for OpaquePaintImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for QueueAuthenticPixels() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for RemapImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for RemapImages() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for RemoveImageArtifact() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for RotationalBlurImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for RotationalBlurImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for SelectiveBlurImageChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for SetImageAlphaChannel() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for SetImageArtifact() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for SetMagickMemoryMethods() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for SparseColorImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for StatisticImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for SyncAuthenticPixels() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for TransformImageColorspace() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for TransparentPaintImage() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for TransparentPaintImageChroma() in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... no
checking for QueryMagickColorname() new signature... yes
checking for Image.type in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for DrawInfo.kerning in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for DrawInfo.interline_spacing in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for DrawInfo.interword_spacing in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for DitherMethod in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for MagickFunction in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for ImageLayerMethod in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for long double in assert.h,ctype.h,stdio.h,stdlib.h,math.h,time.h,stdint.h,sys/types.h,wand/MagickWand.h... yes
checking for AlphaChannelType.CopyAlphaChannel... yes
checking for AlphaChannelType.BackgroundAlphaChannel... yes
checking for CompositeOperator.BlurCompositeOp... yes
checking for CompositeOperator.DistortCompositeOp... yes
checking for CompositeOperator.LinearBurnCompositeOp... yes
checking for CompositeOperator.LinearDodgeCompositeOp... yes
checking for CompositeOperator.MathematicsCompositeOp... yes
checking for CompositeOperator.PegtopLightCompositeOp... yes
checking for CompositeOperator.PinLightCompositeOp... yes
checking for CompositeOperator.VividLightCompositeOp... yes
checking for CompressionType.DXT1Compression... yes
checking for CompressionType.DXT3Compression... yes
checking for CompressionType.DXT5Compression... yes
checking for CompressionType.ZipSCompression... yes
checking for CompressionType.PizCompression... yes
checking for CompressionType.Pxr24Compression... yes
checking for CompressionType.B44Compression... yes
checking for CompressionType.B44ACompression... yes
checking for DistortImageMethod.BarrelDistortion... yes
checking for DistortImageMethod.BarrelInverseDistortion... yes
checking for DistortImageMethod.BilinearForwardDistortion... yes
checking for DistortImageMethod.BilinearReverseDistortion... yes
checking for DistortImageMethod.DePolarDistortion... yes
checking for DistortImageMethod.PolarDistortion... yes
checking for DistortImageMethod.PolynomialDistortion... yes
checking for DistortImageMethod.ShepardsDistortion... yes
checking for DitherMethod.NoDitherMethod... yes
checking for FilterTypes.KaiserFilter... yes
checking for FilterTypes.WelshFilter... yes
checking for FilterTypes.ParzenFilter... yes
checking for FilterTypes.LagrangeFilter... yes
checking for FilterTypes.BohmanFilter... yes
checking for FilterTypes.BartlettFilter... yes
checking for FilterTypes.SentinelFilter... yes
checking for MagickEvaluateOperator.PowEvaluateOperator... yes
checking for MagickEvaluateOperator.LogEvaluateOperator... yes
checking for MagickEvaluateOperator.ThresholdEvaluateOperator... yes
checking for MagickEvaluateOperator.ThresholdBlackEvaluateOperator... yes
checking for MagickEvaluateOperator.ThresholdWhiteEvaluateOperator... yes
checking for MagickEvaluateOperator.GaussianNoiseEvaluateOperator... yes
checking for MagickEvaluateOperator.ImpulseNoiseEvaluateOperator... yes
checking for MagickEvaluateOperator.LaplacianNoiseEvaluateOperator... yes
checking for MagickEvaluateOperator.MultiplicativeNoiseEvaluateOperator... yes
checking for MagickEvaluateOperator.PoissonNoiseEvaluateOperator... yes
checking for MagickEvaluateOperator.UniformNoiseEvaluateOperator... yes
checking for MagickEvaluateOperator.CosineEvaluateOperator... yes
checking for MagickEvaluateOperator.SineEvaluateOperator... yes
checking for MagickEvaluateOperator.AddModulusEvaluateOperator... yes
checking for MagickFunction.ArcsinFunction... yes
checking for MagickFunction.ArctanFunction... yes
checking for MagickFunction.PolynomialFunction... yes
checking for MagickFunction.SinusoidFunction... yes
checking for ImageLayerMethod.FlattenLayer... yes
checking for ImageLayerMethod.MergeLayer... yes
checking for ImageLayerMethod.MosaicLayer... yes
checking for ImageLayerMethod.TrimBoundsLayer... yes
checking for VirtualPixelMethod.HorizontalTileVirtualPixelMethod... yes
checking for VirtualPixelMethod.VerticalTileVirtualPixelMethod... yes
checking for VirtualPixelMethod.HorizontalTileEdgeVirtualPixelMethod... yes
checking for VirtualPixelMethod.VerticalTileEdgeVirtualPixelMethod... yes
checking for VirtualPixelMethod.CheckerTileVirtualPixelMethod... yes
checking for ruby/io.h... yes
checking for rb_frame_this_func() in ruby.h,ruby/io.h... yes
creating extconf.h
creating Makefile


======================================================================
Tue 25Nov14 23:41:19
This installation of RMagick 2.13.4 is configured for
Ruby 2.0.0 (universal.x86_64-darwin14) and ImageMagick 
======================================================================



make "DESTDIR=" clean

make "DESTDIR="
compiling rmagick.c
compiling rmdraw.c
rmdraw.c:315:15: warning: comparison of constant 100 with expression of type 'WeightType' is always true [-Wtautological-constant-out-of-range-compare]
        if (w < 100 || w > 900)
            ~ ^ ~~~
rmdraw.c:315:26: warning: comparison of constant 900 with expression of type 'WeightType' is always false [-Wtautological-constant-out-of-range-compare]
        if (w < 100 || w > 900)
                       ~ ^ ~~~
rmdraw.c:1878:17: warning: 'AllocateImage' is deprecated [-Wdeprecated-declarations]
        image = AcquireImage(info);
                ^
./rmagick.h:176:28: note: expanded from macro 'AcquireImage'
#define AcquireImage(info) AllocateImage(info)
                           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:127:4: note: 'AllocateImage' has been explicitly marked deprecated here
  *AllocateImage(const ImageInfo *) magick_attribute((deprecated)),
   ^
3 warnings generated.
rmdraw.c:315:15: warning: comparison of constant 100 with expression of type 'WeightType' is always true [-Wtautological-constant-out-of-range-compare]
        if (w < 100 || w > 900)
            ~ ^ ~~~
rmdraw.c:315:26: warning: comparison of constant 900 with expression of type 'WeightType' is always false [-Wtautological-constant-out-of-range-compare]
        if (w < 100 || w > 900)
                       ~ ^ ~~~
rmdraw.c:1878:17: warning: 'AllocateImage' is deprecated [-Wdeprecated-declarations]
        image = AcquireImage(info);
                ^
./rmagick.h:176:28: note: expanded from macro 'AcquireImage'
#define AcquireImage(info) AllocateImage(info)
                           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:127:4: note: 'AllocateImage' has been explicitly marked deprecated here
  *AllocateImage(const ImageInfo *) magick_attribute((deprecated)),
   ^
3 warnings generated.
compiling rmenum.c
rmenum.c:450:13: warning: 5 enumeration values not handled in switch: 'DivideSrcCompositeOp', 'MinusSrcCompositeOp', 'DarkenIntensityCompositeOp'... [-Wswitch]
    switch (op)
            ^
rmenum.c:565:13: warning: enumeration values 'LZMACompression', 'JBIG1Compression', and 'JBIG2Compression' not handled in switch [-Wswitch]
    switch (ct)
            ^
rmenum.c:674:12: warning: 9 enumeration values not handled in switch: 'SincFastFilter', 'LanczosSharpFilter', 'Lanczos2Filter'... [-Wswitch]
    switch(type)
           ^
rmenum.c:922:12: warning: 5 enumeration values not handled in switch: 'Average9InterpolatePixel', 'Average16InterpolatePixel', 'BlendInterpolatePixel'... [-Wswitch]
    switch(interpolate)
           ^
4 warnings generated.
rmenum.c:450:13: warning: 5 enumeration values not handled in switch: 'DivideSrcCompositeOp', 'MinusSrcCompositeOp', 'DarkenIntensityCompositeOp'... [-Wswitch]
    switch (op)
            ^
rmenum.c:565:13: warning: enumeration values 'LZMACompression', 'JBIG1Compression', and 'JBIG2Compression' not handled in switch [-Wswitch]
    switch (ct)
            ^
rmenum.c:674:12: warning: 9 enumeration values not handled in switch: 'SincFastFilter', 'LanczosSharpFilter', 'Lanczos2Filter'... [-Wswitch]
    switch(type)
           ^
rmenum.c:922:12: warning: 5 enumeration values not handled in switch: 'Average9InterpolatePixel', 'Average16InterpolatePixel', 'BlendInterpolatePixel'... [-Wswitch]
    switch(interpolate)
           ^
4 warnings generated.
compiling rmfill.c
rmfill.c:143:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:159:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:233:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:243:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:315:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        col_pixels = SetImagePixels(image, (long int)x, 0, 1, image->rows);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:324:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:414:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:430:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:521:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:537:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
10 warnings generated.
rmfill.c:143:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:159:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:233:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:243:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:315:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        col_pixels = SetImagePixels(image, (long int)x, 0, 1, image->rows);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:324:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:414:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:430:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmfill.c:521:22: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        row_pixels = SetImagePixels(image, 0, (long int)y, image->columns, 1);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmfill.c:537:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(image);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
10 warnings generated.
compiling rmilist.c
rmilist.c:459:12: warning: 'MapImages' is deprecated [-Wdeprecated-declarations]
    (void) MapImages(new_images, map, dither);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:212:3: note: 'MapImages' has been explicitly marked deprecated here
  MapImages(Image *,const Image *,const MagickBooleanType)
  ^
rmilist.c:678:20: warning: 'MapImages' is deprecated [-Wdeprecated-declarations]
            (void) MapImages(new_images, NULL, 0);
                   ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:212:3: note: 'MapImages' has been explicitly marked deprecated here
  MapImages(Image *,const Image *,const MagickBooleanType)
  ^
2 warnings generated.
rmilist.c:459:12: warning: 'MapImages' is deprecated [-Wdeprecated-declarations]
    (void) MapImages(new_images, map, dither);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:212:3: note: 'MapImages' has been explicitly marked deprecated here
  MapImages(Image *,const Image *,const MagickBooleanType)
  ^
rmilist.c:678:20: warning: 'MapImages' is deprecated [-Wdeprecated-declarations]
            (void) MapImages(new_images, NULL, 0);
                   ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:212:3: note: 'MapImages' has been explicitly marked deprecated here
  MapImages(Image *,const Image *,const MagickBooleanType)
  ^
2 warnings generated.
compiling rmimage.c
rmimage.c:2784:12: warning: 'ColorFloodfillImage' is deprecated [-Wdeprecated-declarations]
    (void) ColorFloodfillImage(new_image, draw_info, target, x, y, (PaintMethod)fill_method);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:183:3: note: 'ColorFloodfillImage' has been explicitly marked deprecated here
  ColorFloodfillImage(Image *,const DrawInfo *,const PixelPacket,const ssize_t,
  ^
rmimage.c:3680:12: warning: 'SetImageAttribute' is deprecated [-Wdeprecated-declarations]
    (void) SetImageAttribute(comp_image, "[modify-outside-overlay]", "false");
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:229:3: note: 'SetImageAttribute' has been explicitly marked deprecated here
  SetImageAttribute(Image *,const char *,const char *)
  ^
rmimage.c:3919:13: warning: 'AllocateImage' is deprecated [-Wdeprecated-declarations]
    image = AcquireImage(NULL);
            ^
./rmagick.h:176:28: note: expanded from macro 'AcquireImage'
#define AcquireImage(info) AllocateImage(info)
                           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:127:4: note: 'AllocateImage' has been explicitly marked deprecated here
  *AllocateImage(const ImageInfo *) magick_attribute((deprecated)),
   ^
rmimage.c:6993:14: warning: 'AcquireImagePixels' is deprecated [-Wdeprecated-declarations]
    pixels = AcquireImagePixels(image, x, y, columns, rows, exception);
             ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:120:4: note: 'AcquireImagePixels' has been explicitly marked deprecated here
  *AcquireImagePixels(const Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:8167:12: warning: 'MapImage' is deprecated [-Wdeprecated-declarations]
    (void) MapImage(new_image, map, dither);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:210:3: note: 'MapImage' has been explicitly marked deprecated here
  MapImage(Image *,const Image *,const MagickBooleanType)
  ^
rmimage.c:8395:17: warning: 'GetImagePixels' is deprecated [-Wdeprecated-declarations]
            q = GetImagePixels(clip_mask, 0, y, clip_mask->columns, 1);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:264:4: note: 'GetImagePixels' has been explicitly marked deprecated here
  *GetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:8418:13: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
            SyncImagePixels(clip_mask);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmimage.c:8618:12: warning: 'MatteFloodfillImage' is deprecated [-Wdeprecated-declarations]
    (void) MatteFloodfillImage(new_image, target, op, x, y, method);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:214:3: note: 'MatteFloodfillImage' has been explicitly marked deprecated here
  MatteFloodfillImage(Image *,const PixelPacket,const Quantum,const ssize_t,
  ^
rmimage.c:9127:13: warning: 'AllocateImage' is deprecated [-Wdeprecated-declarations]
    image = AcquireImage(info);
            ^
./rmagick.h:176:28: note: expanded from macro 'AcquireImage'
#define AcquireImage(info) AllocateImage(info)
                           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:127:4: note: 'AllocateImage' has been explicitly marked deprecated here
  *AllocateImage(const ImageInfo *) magick_attribute((deprecated)),
   ^
rmimage.c:9382:13: warning: 'PaintOpaqueImageChannel' is deprecated [-Wdeprecated-declarations]
    okay =  PaintOpaqueImageChannel(new_image, DefaultChannels, &target_pp, &fill_pp);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:223:3: note: 'PaintOpaqueImageChannel' has been explicitly marked deprecated here
  PaintOpaqueImageChannel(Image *,const ChannelType,const MagickPixelPacket *,
  ^
rmimage.c:9820:22: warning: 'AcquireImagePixels' is deprecated [-Wdeprecated-declarations]
        old_color = *AcquireImagePixels(image, x, y, 1, 1, exception);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:120:4: note: 'AcquireImagePixels' has been explicitly marked deprecated here
  *AcquireImagePixels(const Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:9832:36: warning: 'GetIndexes' is deprecated [-Wdeprecated-declarations]
            IndexPacket *indexes = GetIndexes(image);
                                   ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:159:4: note: 'GetIndexes' has been explicitly marked deprecated here
  *GetIndexes(const Image *) magick_attribute((deprecated)),
   ^
rmimage.c:9871:13: warning: 'GetImagePixels' is deprecated [-Wdeprecated-declarations]
    pixel = GetImagePixels(image, x, y, 1, 1);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:264:4: note: 'GetImagePixels' has been explicitly marked deprecated here
  *GetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:9889:5: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
    SyncImagePixels(image);
    ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmimage.c:10412:17: warning: 'RadialBlurImage' is deprecated [-Wdeprecated-declarations]
    new_image = RadialBlurImage(image, NUM2DBL(angle), exception);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:144:4: note: 'RadialBlurImage' has been explicitly marked deprecated here
  *RadialBlurImage(const Image *,const double,ExceptionInfo *)
   ^
rmimage.c:10466:17: warning: 'RadialBlurImageChannel' is deprecated [-Wdeprecated-declarations]
    new_image = RadialBlurImageChannel(image, channels, NUM2DBL(argv[0]), exception);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:146:4: note: 'RadialBlurImageChannel' has been explicitly marked deprecated here
  *RadialBlurImageChannel(const Image *,const ChannelType,const double,
   ^
rmimage.c:10739:17: warning: 'RecolorImage' is deprecated [-Wdeprecated-declarations]
    new_image = RecolorImage(image, order, matrix, exception);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:148:4: note: 'RecolorImage' has been explicitly marked deprecated here
  *RecolorImage(const Image *,const size_t,const double *,ExceptionInfo *)
   ^
rmimage.c:12911:18: warning: 'GetImagePixels' is deprecated [-Wdeprecated-declarations]
        pixels = GetImagePixels(image, x, y, cols, rows);
                 ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:264:4: note: 'GetImagePixels' has been explicitly marked deprecated here
  *GetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:12927:13: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
            SyncImagePixels(image);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmimage.c:13101:12: warning: 'ColorFloodfillImage' is deprecated [-Wdeprecated-declarations]
    (void) ColorFloodfillImage(new_image, draw_info, color, x, y, method);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:183:3: note: 'ColorFloodfillImage' has been explicitly marked deprecated here
  ColorFloodfillImage(Image *,const DrawInfo *,const PixelPacket,const ssize_t,
  ^
rmimage.c:13659:12: warning: 'PaintTransparentImage' is deprecated [-Wdeprecated-declarations]
    okay = PaintTransparentImage(new_image, &color, opacity);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:225:3: note: 'PaintTransparentImage' has been explicitly marked deprecated here
  PaintTransparentImage(Image *,const MagickPixelPacket *,const Quantum)
  ^
rmimage.c:14688:13: warning: 'AcquireImagePixels' is deprecated [-Wdeprecated-declarations]
        p = AcquireImagePixels(reflection, 0, y, image->columns, 1, exception);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:120:4: note: 'AcquireImagePixels' has been explicitly marked deprecated here
  *AcquireImagePixels(const Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:14700:13: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        q = SetImagePixels(reflection, 0, y, image->columns, 1);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:14721:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(reflection);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
24 warnings generated.
rmimage.c:2784:12: warning: 'ColorFloodfillImage' is deprecated [-Wdeprecated-declarations]
    (void) ColorFloodfillImage(new_image, draw_info, target, x, y, (PaintMethod)fill_method);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:183:3: note: 'ColorFloodfillImage' has been explicitly marked deprecated here
  ColorFloodfillImage(Image *,const DrawInfo *,const PixelPacket,const ssize_t,
  ^
rmimage.c:3680:12: warning: 'SetImageAttribute' is deprecated [-Wdeprecated-declarations]
    (void) SetImageAttribute(comp_image, "[modify-outside-overlay]", "false");
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:229:3: note: 'SetImageAttribute' has been explicitly marked deprecated here
  SetImageAttribute(Image *,const char *,const char *)
  ^
rmimage.c:3919:13: warning: 'AllocateImage' is deprecated [-Wdeprecated-declarations]
    image = AcquireImage(NULL);
            ^
./rmagick.h:176:28: note: expanded from macro 'AcquireImage'
#define AcquireImage(info) AllocateImage(info)
                           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:127:4: note: 'AllocateImage' has been explicitly marked deprecated here
  *AllocateImage(const ImageInfo *) magick_attribute((deprecated)),
   ^
rmimage.c:6993:14: warning: 'AcquireImagePixels' is deprecated [-Wdeprecated-declarations]
    pixels = AcquireImagePixels(image, x, y, columns, rows, exception);
             ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:120:4: note: 'AcquireImagePixels' has been explicitly marked deprecated here
  *AcquireImagePixels(const Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:8167:12: warning: 'MapImage' is deprecated [-Wdeprecated-declarations]
    (void) MapImage(new_image, map, dither);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:210:3: note: 'MapImage' has been explicitly marked deprecated here
  MapImage(Image *,const Image *,const MagickBooleanType)
  ^
rmimage.c:8395:17: warning: 'GetImagePixels' is deprecated [-Wdeprecated-declarations]
            q = GetImagePixels(clip_mask, 0, y, clip_mask->columns, 1);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:264:4: note: 'GetImagePixels' has been explicitly marked deprecated here
  *GetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:8418:13: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
            SyncImagePixels(clip_mask);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmimage.c:8618:12: warning: 'MatteFloodfillImage' is deprecated [-Wdeprecated-declarations]
    (void) MatteFloodfillImage(new_image, target, op, x, y, method);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:214:3: note: 'MatteFloodfillImage' has been explicitly marked deprecated here
  MatteFloodfillImage(Image *,const PixelPacket,const Quantum,const ssize_t,
  ^
rmimage.c:9127:13: warning: 'AllocateImage' is deprecated [-Wdeprecated-declarations]
    image = AcquireImage(info);
            ^
./rmagick.h:176:28: note: expanded from macro 'AcquireImage'
#define AcquireImage(info) AllocateImage(info)
                           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:127:4: note: 'AllocateImage' has been explicitly marked deprecated here
  *AllocateImage(const ImageInfo *) magick_attribute((deprecated)),
   ^
rmimage.c:9382:13: warning: 'PaintOpaqueImageChannel' is deprecated [-Wdeprecated-declarations]
    okay =  PaintOpaqueImageChannel(new_image, DefaultChannels, &target_pp, &fill_pp);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:223:3: note: 'PaintOpaqueImageChannel' has been explicitly marked deprecated here
  PaintOpaqueImageChannel(Image *,const ChannelType,const MagickPixelPacket *,
  ^
rmimage.c:9820:22: warning: 'AcquireImagePixels' is deprecated [-Wdeprecated-declarations]
        old_color = *AcquireImagePixels(image, x, y, 1, 1, exception);
                     ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:120:4: note: 'AcquireImagePixels' has been explicitly marked deprecated here
  *AcquireImagePixels(const Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:9832:36: warning: 'GetIndexes' is deprecated [-Wdeprecated-declarations]
            IndexPacket *indexes = GetIndexes(image);
                                   ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:159:4: note: 'GetIndexes' has been explicitly marked deprecated here
  *GetIndexes(const Image *) magick_attribute((deprecated)),
   ^
rmimage.c:9871:13: warning: 'GetImagePixels' is deprecated [-Wdeprecated-declarations]
    pixel = GetImagePixels(image, x, y, 1, 1);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:264:4: note: 'GetImagePixels' has been explicitly marked deprecated here
  *GetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:9889:5: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
    SyncImagePixels(image);
    ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmimage.c:10412:17: warning: 'RadialBlurImage' is deprecated [-Wdeprecated-declarations]
    new_image = RadialBlurImage(image, NUM2DBL(angle), exception);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:144:4: note: 'RadialBlurImage' has been explicitly marked deprecated here
  *RadialBlurImage(const Image *,const double,ExceptionInfo *)
   ^
rmimage.c:10466:17: warning: 'RadialBlurImageChannel' is deprecated [-Wdeprecated-declarations]
    new_image = RadialBlurImageChannel(image, channels, NUM2DBL(argv[0]), exception);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:146:4: note: 'RadialBlurImageChannel' has been explicitly marked deprecated here
  *RadialBlurImageChannel(const Image *,const ChannelType,const double,
   ^
rmimage.c:10739:17: warning: 'RecolorImage' is deprecated [-Wdeprecated-declarations]
    new_image = RecolorImage(image, order, matrix, exception);
                ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:148:4: note: 'RecolorImage' has been explicitly marked deprecated here
  *RecolorImage(const Image *,const size_t,const double *,ExceptionInfo *)
   ^
rmimage.c:12911:18: warning: 'GetImagePixels' is deprecated [-Wdeprecated-declarations]
        pixels = GetImagePixels(image, x, y, cols, rows);
                 ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:264:4: note: 'GetImagePixels' has been explicitly marked deprecated here
  *GetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:12927:13: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
            SyncImagePixels(image);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
rmimage.c:13101:12: warning: 'ColorFloodfillImage' is deprecated [-Wdeprecated-declarations]
    (void) ColorFloodfillImage(new_image, draw_info, color, x, y, method);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:183:3: note: 'ColorFloodfillImage' has been explicitly marked deprecated here
  ColorFloodfillImage(Image *,const DrawInfo *,const PixelPacket,const ssize_t,
  ^
rmimage.c:13659:12: warning: 'PaintTransparentImage' is deprecated [-Wdeprecated-declarations]
    okay = PaintTransparentImage(new_image, &color, opacity);
           ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:225:3: note: 'PaintTransparentImage' has been explicitly marked deprecated here
  PaintTransparentImage(Image *,const MagickPixelPacket *,const Quantum)
  ^
rmimage.c:14688:13: warning: 'AcquireImagePixels' is deprecated [-Wdeprecated-declarations]
        p = AcquireImagePixels(reflection, 0, y, image->columns, 1, exception);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:120:4: note: 'AcquireImagePixels' has been explicitly marked deprecated here
  *AcquireImagePixels(const Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:14700:13: warning: 'SetImagePixels' is deprecated [-Wdeprecated-declarations]
        q = SetImagePixels(reflection, 0, y, image->columns, 1);
            ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:271:4: note: 'SetImagePixels' has been explicitly marked deprecated here
  *SetImagePixels(Image *,const ssize_t,const ssize_t,const size_t,
   ^
rmimage.c:14721:9: warning: 'SyncImagePixels' is deprecated [-Wdeprecated-declarations]
        SyncImagePixels(reflection);
        ^
/opt/local/include/ImageMagick-6/magick/deprecate.h:232:3: note: 'SyncImagePixels' has been explicitly marked deprecated here
  SyncImagePixels(Image *) magick_attribute((deprecated)),
  ^
24 warnings generated.
compiling rminfo.c
compiling rmmain.c
rmmain.c:1724:28: error: use of undeclared identifier 'MagickSupport'
    features = rb_str_new2(MagickSupport);
                           ^
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/ruby/intern.h:760:27: note: expanded from macro 'rb_str_new_cstr'
    (__builtin_constant_p(str)) ?               \
                          ^
rmmain.c:1724:28: error: use of undeclared identifier 'MagickSupport'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/ruby/intern.h:761:14: note: expanded from macro 'rb_str_new_cstr'
        rb_str_new((str), (long)strlen(str)) :  \
                    ^
rmmain.c:1724:28: error: use of undeclared identifier 'MagickSupport'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/ruby/intern.h:762:18: note: expanded from macro 'rb_str_new_cstr'
        rb_str_new_cstr(str);                   \
                        ^
rmmain.c:1724:14: error: assigning to 'volatile VALUE' (aka 'volatile unsigned long') from incompatible type 'void'
    features = rb_str_new2(MagickSupport);
             ^
4 errors generated.
make: *** [rmmain.o] Error 1

make failed, exit code 2

Gem files will remain installed in /Library/Ruby/Gems/2.0.0/gems/rmagick-2.13.4 for inspection.
Results logged to /Library/Ruby/Gems/2.0.0/extensions/universal-darwin-14/2.0.0/rmagick-2.13.4/gem_make.out
An error occurred while installing rmagick (2.13.4), and Bundler cannot continue.
Make sure that `gem install rmagick -v '2.13.4'` succeeds before bundling.
```

--- a/ext/RMagick/rmmain.c
+++ b/ext/RMagick/rmmain.c
@@ -1719,9 +1719,16 @@ features_constant(void)
     volatile VALUE features;
 
 #if defined(HAVE_GETMAGICKFEATURES)
+    // 6.5.7 - latest (7.0.0)
     features = rb_str_new2(GetMagickFeatures());
-#else
+#elif defined(MagickFeatures)
+    // 6.5.7 - latest (7.0.0)
+    features = rb_str_new2(MagickFeatures);
+#elif defined(MagickSuuport)
+    // 6.5.5 - 6.5.6
     features = rb_str_new2(MagickSupport);
+#else
+    features = rb_str_new("unknown",7);
 #endif
 
     rb_obj_freeze(features);

